addons/website_event/models/event_event.py

```diff
@@ -507,6 +507,13 @@ def _track_subtype(self, init_values):
             return self.env.ref('website_event.mt_event_unpublished', raise_if_not_found=False)
         return super(Event, self)._track_subtype(init_values)
 
+    def _get_external_description(self):
+        # Adding the URL of the event into the description
+        max_length = 2000
+        event_url = f'<a href="{self.get_base_url()}/event/{slug(self)}/register">{self.name}</a>'
+        description = event_url + '\n' + super()._get_external_description()
+        return (description[:max_length] + '...') if len(description) > max_length else description
```

Comment:
Seems a bit overkill to do that whole thing twice.

Would say we either reduce the limit in the other method to 1900 to leave space for that extra URL or live with the fact that it will be 2000 + url size.

Depends a bit on the hard limit.